src/platform/utilities/oauth/utilities.js

```diff
@@ -216,14 +216,14 @@ export const requestToken = async ({ code, redirectUri, csp }) => {
   return response;
 };
 
-export const refresh = async () => {
+export const refresh = async ({ type }) => {
```

Comment:
I'm curious if there's any particular reason why this is an object rather than just a single parameter named `type`.

Reply:
Hey Ryan, no particular reason for type to be destructured, was just helpful when I was adding new functionality. I appreciate the question and will make sure to keep it DRYer next time.